CRM/Civicase/APIHelpers/CustomGroups.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * Custom Groups API Helper Class.
+ */
+class CRM_Civicase_APIHelpers_CustomGroups {
+
+  /**
+   * Returns a list of all active custom groups.
+   *
+   * @return array
+   *   Custom Group Api response.
+   */
+  public static function getAllActiveGroups() {
+    return civicrm_api3('CustomGroup', 'get', [
+      'extends' => $params['entity_type'],
```

Comment:
$param us defined in this function but I don't see it passed in the function. Can we modify it so that it accepts two parameters. `$entityType` and `$options`

Reply:
Sorry @tunbola , there was a bit of a mistake here. We did need to pass `$params['entity_type']`, but not the limit. We actually need ALL the groups. We need all groups and passing a limit won't work as we might think because the group might not even have values defined for the given record. So if I pass a limit of 1, I can get the `My Custom Case Fields` group, but that group might not have values for the case ID 123.

Since we are combining the results from 2 api calls, the limit does not work. The goal of this endpoint is not to limit the results either, but to display all of them.